buf.yaml

```diff
@@ -0,0 +1,14 @@
+version: v1beta1
+build:
+  roots:
+    - proto
+lint:
+  use:
+    - DEFAULT
+  except:
```

Comment:
rather than globally turning these off, can you make these specific to the config file?
```
  ignore_only:
    ENUM_VALUE_PREFIX:
      - proto/hypertrace/agent/config/v1/config.proto
    ENUM_ZERO_VALUE_SUFFIX:
      - proto/hypertrace/agent/config/v1/config.proto
```